utils/gha-runners/get-system-info.sh

```diff
@@ -13,8 +13,8 @@ function system_info {
 	echo "libndctl: $(pkg-config --modversion libndctl || echo 'libndctl not found')"
 	echo "valgrind: $(pkg-config --modversion valgrind || echo 'valgrind not found')"
 	echo "******************** memory-info *******************"
-	ipmctl show -dimm || true
-	ipmctl show -topology || true
+	sudo ipmctl show -dimm || true
+	sudo ipmctl show -topology || true
```

Comment:
I don't think we want to add sudo to the calls in scripts we run on internal runners.
I'd rather remove that call if it doesn't work.

Reply:
yes, we shouldn't be using `sudo` inside of scripts.